less/transferwise.less

```diff
@@ -56,7 +56,7 @@ a.btn {
 }
 
 .nav > li > a {
-    padding: 11px 16px 9px;
+    padding: 11px 16px 5px;
```

Comment:
Very important rule is to have hight exactly 40px but we have 44px all over the project. And i've changed bottom padding to make icon and text look more in centre